src/component/variantPage/functionalPrediction/AlphaMissense.tsx

```diff
@@ -0,0 +1,232 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { DefaultTooltip } from 'cbioportal-frontend-commons';
+import { action, makeObservable, observable } from 'mobx';
+import { observer } from 'mobx-react';
+import { Collapse, Table } from 'react-bootstrap';
+import functionalImpactColor from './styles/functionalImpactTooltip.module.scss';
+import functionalGroupsStyle from '../functionalGroups.module.scss';
+
+export interface IAlphamissenseProps {
+    amClass?: string;
+    amPathogenicityScore?: number;
+}
+
+const ALPHAMISSENSE_URL = 'https://www.science.org/doi/10.1126/science.adg7492';
+
+const AlphamissenseInfo: React.FunctionComponent = () => {
+    return (
+        <div>
+            <a
+                href={ALPHAMISSENSE_URL}
+                target="_blank"
+                rel="noopener noreferrer"
+            >
+                AlphaMissense
+            </a>{' '}
+            This is AlphaMissense information
```

Comment:
We need to display an actual info here.

Reply:
Please add a short description of what is Alpha Missense (1-2 sentences), and briefly describe the pathogenicity classes and score range of each group, also add link to Alpha Missense text to the paper

Reply:
added it